platforms/documentation/docs/src/docs/userguide/authoring-builds/gradle-properties/service_injection.adoc

```diff
@@ -33,6 +33,7 @@ The following services are available for injection in all Gradle instantiated ob
 3. <<filesystemoperations,`FileSystemOperations`>> - Allows a task to run operations on the filesystem such as deleting files, copying files or syncing directories.
 4. <<archiveoperations,`ArchiveOperations`>> - Allows a task to run operations on archive files such as ZIP or TAR files.
 5. <<execoperations,`ExecOperations`>> - Allows a task to run external processes with dedicated support for running external `java` programs.
+6. <<dependencyfactory,`DependencyFactory`>> - Allows a plugin to create dependencies in a type-safe way.
```

Comment:
This isn't available in workers, so it should be put in the project and settings sections below